packages/playground/nested-deps/vite.config.js

```diff
@@ -1,15 +1,23 @@
+const path = require('path')
+const packageFPath = path.resolve(__dirname, 'test-package-f')
 /**
  * @type {import('vite').UserConfig}
  */
 module.exports = {
+  resolve: {
+    alias: {
+      '{{ F }}': packageFPath
+    }
```

Comment:
Just to test something out, could you test:

```suggestion
    alias: [
      { find: '{{ F }}', replacement: packageFPath }
    ]
```

Just want to proof a point and maybe we have a bug here 🤔